src/Instance/InstanceDetails/InstanceContributorsView/InstanceContributorsView.js

```diff
@@ -72,6 +77,46 @@ const InstanceContributorsView = ({
     );
   }, [contributors, contributorTypes, contributorNameTypes]);
 
+  const getName = (item) => {
+    if (segment === segments.instances && source === 'MARC' && 'item.authorityId') {
+      return (
+        <>
```

Comment:
Can we make this element a separate component? It's just mostly a duplicate of https://github.com/folio-org/ui-inventory/pull/1794/files

Reply:
Resolving the comment - DOM structure inside InstancesList doesn't allow for this same component